config/database.yml

```
@@ -7,7 +7,7 @@ default: &default
 
 development:
   <<: *default
-  database: goodcity_server_development
+  database: goodcity_test
```

just catching this @patrixr 
```suggestion
  database: goodcity_server_development
```

---

whoops